site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,171 @@
+Additionally, users with 2 cores, and either 2 or 4 threads, should not enable multicore as it will most likely result in a performance 
+loss for them due to the lack of physical cores on their CPUs. Our [hardware recommendations](https://yuzu-emu.org/help/quickstart/#hardware) 
+have been updated accordingly.
+
+## Unreal Engine fixes
+
+[Rodrigo](https://github.com/ReinUsesLisp) implemented [rendering more than one slice of 3D textures](https://github.com/yuzu-emu/yuzu/pull/4027), fixing the most glaring 
+issue in Unreal Engine 4 games, known as “the Rainbow”. This also improves the previous implementation that was used in 
+Xenoblade games.
+
+{{< single-title-imgs
+    "Your Excellency (OCTOPATH TRAVELER)"
+    "./01.png"
+    "./02.png"
+  >}}
+
+## Animal Crossing: New Horizons changes
+
+[Rodrigo](https://github.com/ReinUsesLisp) fixed `Animal Crossing: New Horizons` terrain borders in Vulkan by implementing 
+[constant attributes](https://github.com/yuzu-emu/yuzu/pull/3930). This is not a native extension, constant attributes have to be emulated in Vulkan as there is currently no official support for it.
+
+{{< single-title-imgs
+    "Beautiful beaches, now in Vulkan too (Animal Crossing: New Horizons)"
+    "./04.png"
+    "./05.png"
+  >}}
+
+[bunnei](https://github.com/bunnei) implemented [time zone support](https://github.com/yuzu-emu/yuzu/pull/3909), and Windows users will find that yuzu automatically detects their time zone. For those not on Windows (or want to spice up their life), you can manually change your system time via the "Custom RTC" option in the System settings. Previously, yuzu always assumed the user was located in the GMT+0 time zone.
+
+[bunnei](https://github.com/bunnei) also improved the [saving mechanism](https://github.com/yuzu-emu/yuzu/pull/3665). Most games save their data in each user profile, but `Animal Crossing: New Horizons` does it via a “device” profile, so yuzu had to accommodate for that.
+
+## Xenoblade specific fixes
+
+Rendering bugs are abundant in Xenoblade games, due to the complexity of their engine, they are not trivial to solve. 
+However, with the help of [gdkchan](https://github.com/gdkchan), and using this [Pull Request](https://github.com/Ryujinx/Ryujinx/pull/1277) from [Ryujinx](https://github.com/Ryujinx/Ryujinx), [Rodrigo](https://github.com/ReinUsesLisp) fixed one of the major rendering issues in `Xenoblade Chronicles 2` related to [front face flipping](https://github.com/yuzu-emu/yuzu/pull/3996). Additional [improvements to texture depth samplings](https://github.com/yuzu-emu/yuzu/pull/3991) resolved some rendering glitches, found for example in clouds and the start menu. Additionally, a better handling of [mipmap overlaps](https://github.com/yuzu-emu/yuzu/pull/4012) solved the constantly moving textures the games previously had. You can see the results below.
```

Comment:
Change `some rendering glitches, found for example in clouds and the start menu.`

to

`some rendering glitches such as the clouds and start menu.`